tests/rptest/tests/audit_log_test.py

```diff
@@ -1231,6 +1237,70 @@ def authz_api_filter_function(service_name, username: str, record):
             'name'] == service_name and record['actor']['user'][
                 'name'] == username
 
+    @cluster(num_nodes=5)
+    def test_excluded_principal(self):
+        """
+        Verifies that principals excluded will not generate audit messages
+        """
+        self.setup_cluster()
+        user2 = "ignored_user"
+        user2_pw = "ignored_user"
+        user2_alg = "SCRAM-SHA-256"
+
+        self.modify_audit_excluded_principals([user2])
+
+        self.admin.create_user(user2, user2_pw, user2_alg)
+        self.super_rpk.sasl_allow_principal(
+            principal=user2,
+            operations=['all'],
+            resource='topic',
+            resource_name='*',
+            username=self.redpanda.SUPERUSER_CREDENTIALS[0],
+            password=self.redpanda.SUPERUSER_CREDENTIALS[1],
+            mechanism=self.redpanda.SUPERUSER_CREDENTIALS[2])
+
+        user2_rpk = self.get_rpk_credentials(username=user2,
+                                             password=user2_pw,
+                                             mechanism=user2_alg)
+
+        _ = self.rpk.list_topics()
+        _ = user2_rpk.list_topics()
+
+        def contains_principal(principal: str, record):
+            if record['class_uid'] == 3002:
+                return record['user']['name'] == principal
+            elif record['class_uid'] == 6003:
+                return record['actor']['user']['name'] == principal
+            return False
+
+        records = self.find_matching_record(
+            lambda record: contains_principal(self.username, record),
+            lambda record_count: record_count > 0,
+            f'Should contain {self.username}')
+
+        assert len(
+            records
+        ) > 0, f'Did not receive any audit messages for principal {self.username}'
+
+        try:
+            records = self.find_matching_record(
```

Comment:
Maybe it would be good to perform some other action with user2, just to know its working, maybe make a `describe` request and have that disabled so it won't be an audit-able event.